src/main/java/net/dv8tion/jda/core/entities/impl/JDAImpl.java

```diff
@@ -44,10 +44,7 @@
 
 import javax.security.auth.login.LoginException;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
+import java.util.*;
```

Comment:
Wildcard came from the use of `Arrays` which set the value to 5, thus it wildcarded.